KVMarketSquare.xcodeproj/xcshareddata/xcschemes/KVMarketSquare.xcscheme

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
theres gotta be a way to tell github to ignore these, i know we can gitignore them locally but would be better here 🤷